src/test/java/io/wisetime/connector/jira/JiraDaoTest.java

```diff
@@ -183,6 +183,11 @@ void findIssuesOrderedById() {
     assertThat(jiraDao.findIssuesOrderedById(0, 100))
         .as("Should be able retrieve matching issue")
         .containsExactlyElementsOf(savedIssues);
+
```

Comment:
Can we also add a negative scenario? Passing a project key that doesn't match `savedIssues`

Reply:
Done